src/app/main/component/user/components/habit/add-new-habit/habit-invite-friends/habit-invite-friends-pop-up/habit-invite-friends-pop-up.component.html

```diff
@@ -67,7 +67,7 @@
     <button class="btn btn-outline-success" mat-dialog-close>
       {{ 'user.habit.invite.btn.cancel' | translate }}
     </button>
-    <button class="btn btn-success" [mat-dialog-close]="true" cdkFocusInitial (click)="setAddedFriends()">
+    <button class="btn btn-success" [mat-dialog-close]="true" cdkFocusInitial (click)="setAddedFriends()" [disabled]="!isAnyFriendSelected">
```

Comment:
Attribute cdkFocusInitial is not allowed here

Reply:
update